libraries/AP_TemperatureSensor/AP_MCP9600.h

```diff
@@ -0,0 +1,186 @@
+/*
```

Comment:
before we add this we will need to do a frontend/backend split in AP_TemperatureSensor, like we have done for other driver types, so we have a TSENS_TYPE parameter to select which type of temperature sensor to enable